rpc-tests/tests/0001-balance-xcm-asset.js

```diff
@@ -0,0 +1,25 @@
+async function run(nodeName, networkInfo, args) {
+    networkInfo.nodesByName["astar"].hrmpTo = 2007;
+    networkInfo.nodesByName["shiden"].hrmpTo = 2006;
+
+    const { wsUri, userDefinedTypes, hrmpTo } =
+        networkInfo.nodesByName[nodeName];
+    const api = await zombie.connect(wsUri, userDefinedTypes);
+
+    const BN = require("bn.js");
+    const ONE = new BN(10).pow(new BN(18));
+
+    await zombie.util.cryptoWaitReady();
+
+    const assetsAsset = await api.query.assets.asset(hrmpTo);
+    const supply = parseInt(JSON.parse(assetsAsset).supply, 16);
+    console.log('assetsAsset.supply', supply);
```

Comment:
Should these printouts be removed?

Reply:
Good question, do you fell they add context or just clutter the output?

Reply:
Personally, it seems like clutter to me :slightly_smiling_face: